src/contrib/cluster/Akka.Cluster.Sharding/ShardRegion.cs

```diff
@@ -323,95 +368,60 @@ internal static Props Props(string typeName, Func<string, Props> entityProps, Cl
         /// <param name="coordinatorPath">TBD</param>
         /// <param name="extractEntityId">TBD</param>
         /// <param name="extractShardId">TBD</param>
-        /// <param name="replicator"></param>
-        /// <param name="majorityMinCap"></param>
         /// <returns>TBD</returns>
-        internal static Props ProxyProps(string typeName, ClusterShardingSettings settings, string coordinatorPath, ExtractEntityId extractEntityId, ExtractShardId extractShardId, IActorRef replicator, int majorityMinCap)
+        internal static Props ProxyProps(
+            string typeName,
+            ClusterShardingSettings settings,
+            string coordinatorPath,
+            ExtractEntityId extractEntityId,
+            ExtractShardId extractShardId)
         {
-            return Actor.Props.Create(() => new ShardRegion(typeName, null, settings, coordinatorPath, extractEntityId, extractShardId, PoisonPill.Instance, replicator, majorityMinCap)).WithDeploy(Deploy.Local);
+            return Actor.Props.Create(() => new ShardRegion(
+                typeName,
+                null,
+                settings,
+                coordinatorPath,
+                extractEntityId,
+                extractShardId,
+                PoisonPill.Instance,
+                null))
+                .WithDeploy(Deploy.Local);
         }
 
-        /// <summary>
-        /// TBD
-        /// </summary>
-        public readonly string TypeName;
-        /// <summary>
-        /// TBD
-        /// </summary>
-        public readonly Func<string, Props> EntityProps;
-        /// <summary>
-        /// TBD
-        /// </summary>
-        public readonly ClusterShardingSettings Settings;
-        /// <summary>
-        /// TBD
-        /// </summary>
-        public readonly string CoordinatorPath;
-        /// <summary>
-        /// TBD
-        /// </summary>
-        public readonly ExtractEntityId ExtractEntityId;
-        /// <summary>
-        /// TBD
-        /// </summary>
-        public readonly ExtractShardId ExtractShardId;
-        /// <summary>
-        /// TBD
-        /// </summary>
-        public readonly object HandOffStopMessage;
-
-        private readonly IActorRef _replicator;
-        private readonly int _majorityMinCap;
+        private readonly string TypeName;
```

Comment:
Please read our [contributor guideline](https://getakka.net/community/contributor-guidelines.html) on how to do naming.
the [Scala to C# Conversion Guide](https://github.com/akkadotnet/akka.net/wiki/Scala-to-C%23-Conversion-Guide) can also be helpful.

Reply:
will rename, looks like unfinished refactoring here